DigitalLearningSolutions.Web/Views/Supervisor/Shared/_StaffDetails.cshtml

```diff
@@ -12,6 +12,17 @@
   <dd class="nhsuk-summary-list__actions">
   </dd>
   </div>
+  @if (Model.DelegateIsSupervisor)

```

Comment:
This should be checking if the logged in user is a Supervisor rather than the delegate in the model. The controller should have references to this role and it may need passing to the ViewModel in order to determine visibility of PRN.